### ISSUE TITLE
Add text & background color to draw_string()

### ISSUE THREAD

**Issue by DSchndr** [numworks/epsilon#960]
#### Problem you'd like to fix
Parameters for the text & background color of the string from the draw_string() function from the python kandinsky module don't exsist.

#### Screenshots
/

#### Describe the solution you'd like
Add 2 additional parameters to draw_string() to set the colors of the string.

#### Describe alternatives you've considered
Setting the (background) color manually.
Due to font antialiasing / hinting this looks quite "shitty"

```
def bg(xa,ya,xb,yb,c):
    for x in range(xa,xb):
      for y in range(ya,yb):
        a=get_pixel(x,y)
        if (a==color(255,255,255)):
          set_pixel(x,y,c)
```

**Comment by Ecco** [comment]
Fixed by #961

**Comment by DSchndr** [comment]
Closing since #961 has been merged.
